scripts/Manifest

```diff
@@ -5,7 +5,7 @@ ROOK_VERSION=1.0.4
 CONTOUR_VERSION=0.14.0
 REGISTRY_VERSION=2.7.1
 PROMETHEUS_VERSION=0.33.0
-KOTSADM_VERSION=1.12.1
+KOTSADM_VERSION=alpha
```

Comment:
was this supposed to change? What does it affect?

I just want to make sure we don't make alpha the default somehow

Reply:
This is used in dev to select addons without having to run the kurl web server.